src/pages/vaults/VaultLockedSharesTable.tsx

```diff
@@ -0,0 +1,121 @@
+import { useMemo, useState } from 'react';
+
+import { sum } from 'lodash';
+import styled from 'styled-components';
+
+import { VaultShareUnlock } from '@/constants/abacus';
+import { ButtonShape, ButtonSize } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useApiState } from '@/hooks/useApiState';
+import { useStringGetter } from '@/hooks/useStringGetter';
+import { useLoadedVaultAccount } from '@/hooks/vaultsHooks';
+
+import { tradeViewMixins } from '@/styles/tradeViewMixins';
+
+import { Button } from '@/components/Button';
+import { Output, OutputType } from '@/components/Output';
+import { ColumnDef, Table } from '@/components/Table';
+
+import { runIf } from '@/lib/do';
+
+export const VaultLockedSharesCard = ({ className }: { className?: string }) => {
+  const stringGetter = useStringGetter();
+  const [showShares, setShowShares] = useState(false);
+  const vaultAccount = useLoadedVaultAccount().data;
+  const rawLockedShares = vaultAccount?.vaultShareUnlocks;
+  const lockedShares = useMemo(() => rawLockedShares?.toArray(), [rawLockedShares]);
+  const lockedSharesTotalValue = useMemo(
+    () => sum(lockedShares?.map((s) => s.amountUsdc)),
```

Comment:
does `sum(undefined)` actually work correctly here

Reply:
yeah it's just 0